base/applications/mspaint/history.cpp

```diff
@@ -44,8 +44,9 @@ ImageModel::ImageModel()
 
 void ImageModel::CopyPrevious()
 {
+    DPRINT("CopyPrevious: %d\n", currInd);
```

Comment:
That's ok; also, you could use `__FUNCTION__` if you don't want to hardcode the name of the function.

Reply:
OK.